src/android/java/com/pspdfkit/cordova/action/BasicAction.java

```diff
@@ -28,7 +28,7 @@ public String getName() {
   }
 
   @NonNull
-  protected PSPDFKitCordovaPlugin getPlugin() {
+  public PSPDFKitCordovaPlugin getPlugin() {
```

Comment:
Why is this now `public`? Is this really required? (I can't find its usage)

Making the accessor public increases coupling, as it adds the "provide plugin to anybody" responsibility to the `BasicAction` class, which is something the class should not do.

Reply:
Forgot to clean-up when troubleshooting. It is back to `protected`.